src/lib/tabs/tab-header.ts

```diff
@@ -143,6 +146,11 @@ export class MdTabHeader extends _MdTabHeaderMixinBase
               private _changeDetectorRef: ChangeDetectorRef,
               @Optional() private _dir: Directionality) {
     super();
+
+    // TODO: Add library level window listener https://goo.gl/FJWhZM
```

Comment:
I think you got the wrong comment in your shortlink
(probably meant https://github.com/angular/material2/pull/6304#discussion_r131703706)